package iot.database.exceptions;

public class AlreadyExistsGoodInProviderException extends Exception {
}
